master-firmware/src/robot_helpers/math_helpers.c

```diff
@@ -12,3 +15,24 @@ float angle_delta(float start, float end)
         return res_conj;
     }
 }
+
+bool math_point_is_in_square(poly_t* square, int x, int y)
+{
```

Comment:
There is already `is_in_poly` in `polygon.h`, what is the reason for this function ? Or at least it could be a wrapper around `is_in_poly`.